docs/cli.md

```diff
@@ -45,16 +45,16 @@ This command should also check that the URL is accessible by the Leader node. It
 **`nexus tool validate --on-chain <ident>`**
 
 {% hint style="warning" %}
-The specific design for onchain tools is still in progress and as a result the implementation is not yet implemented. When running the command, it will panic. 
+The specific design for onchain tools is still in progress and as a result the implementation is not yet implemented. When running the command, it will panic.
```

Comment:
_implementation_ is not yet _implemented_ ...

Reply:
went with "missing" https://github.com/Talus-Network/nexus-sdk/pull/148/commits/302c696e5adf443a46675fc53221c75dd50e199f

Reply:
"gone missing"

Reply:
"not yet present" https://github.com/Talus-Network/nexus-sdk/pull/148/commits/55ddfed351425343ef4f844ade048f987b69db3b